README.md

```diff
@@ -382,22 +385,23 @@ spec:
         name: myapp
       metric:
         name: requests-per-second
+        selector:
+          matchLabels:
+            backend: backend1 # optional backend
       target:
         averageValue: "10"
         type: AverageValue
 ```
 
 ### Metric weighting based on backend
 
-Skipper supports sending traffic to different backend based on annotations
-present on the `Ingress` object, or weights on the RouteGroup backends. When
-the metric name is specified without a backend as `requests-per-second` then
-the number of replicas will be calculated based on the full traffic served by
-that ingress/routegroup.  If however only the traffic being routed to a
-specific backend should be used then the backend name can be specified as a
-metric name like `requests-per-second,backend1` which would return the
-requests-per-second being sent to the `backend1`. The ingress annotation where
-the backend weights can be obtained can be specified through the flag
+Skipper supports sending traffic to different backends based on annotations
+present on the `Ingress` object, or weights on the RouteGroup backends. By
+default the number of replicas will be calculated based on the full traffic
+served by that ingress/routegroup.  If however only the traffic being routed to
+a specific backend should be used then the backend name can be specified via
+the `backend` label under `matchLabels` for the metric.  The ingress annotation
+where the backend weights can be obtained can be specified through the flag
```

Comment:
Did you think about adding at least a deprecation message for the old format? I'm not sure if it's worth it, but at least to people understand when they come to see their config that they could/should migrate to the new format. At least I understand the old format is still valid for now.

Reply:
I wouldn't put too much history in docs as it's usually confusing to newcomers. I would put it in the release notes pointing to this PR explaining what the change is.